stylus/components/alerts.styl

```diff
@@ -7,61 +7,68 @@
  * This file contains all needs for alerts
  */
 
-$alerts
-    alert-width = 40rem
-    .coz-alert
-        position          fixed
-        z-index           $alert-index-mobile
-        top               auto
-        right             0
-        bottom            3rem
-        left              0
-        display           flex
-        justify-content   space-between
-        align-items       center
-        box-sizing        border-box
-        box-shadow        0 .375rem 1.125rem 0 rgba(50, 54, 63, .23)
-        background-color  emerald
-        padding           .8rem 1rem
-        color             white
-        opacity           1
-        transition        transform .5s ease-out, opacity .2s ease-out
-        cursor            pointer
+alert-width = 40rem
 
-        p
-            margin      0
-            line-height 1.5
+$alert
+    position fixed
+    z-index $alert-index-mobile
+    top auto
+    right 0
+    bottom 3rem
+    left 0
+    display block
+    color white
+    opacity 1
+    transition transform .5s ease-out, opacity .2s ease-out
+    cursor pointer
 
-        .coz-alert-title
-            font-weight bold
+    @media all and (min-width alert-width)
+        z-index $alert-index
+        top 1rem
+        bottom auto
+        text-align center
 
-        &.coz-alert--hidden
-            transform translateY(5rem)
-            opacity                    0
-            transition-timing-function ease-in
+$alert-wrapper
+    display inline-flex
+    flex-wrap nowrap
+    align-items center
+    justify-content center
+    box-sizing border-box
+    width 100%
+    box-shadow 0 .375rem 1.125rem 0 rgba(50, 54, 63, .23)
```

Comment:
I find the shadow too blurry.

Reply:
Isn't it because there's two of them?

Reply:
Maybe, I have not checked but I know that blurry shadows tend not to perform well on mobile. It can be done in a next PR.